### ISSUE TITLE
possible to "leave" an user

### ISSUE THREAD

**Issue by ineeee** [squidowl/halloy#117]
when opening the context menu on the channel list (via right click), its possible to leave a nickname.

in other words, the menu option to PART a channel also shows up for people, but i can't actually PART someone.

**Comment by ineeee** [comment]
![image](https://github.com/squidowl/halloy/assets/10874672/1193d194-215d-4726-8639-8d69a1426e09)

**Comment by tarkah** [comment]
Yup, no part is sent for these. This is why we called it "leave". All it does it closes the backing history file and removes if from the sidebar so the entry is no longer there.

**Comment by ineeee** [comment]
ah i see, however it seems strange leaving a nick. perhaps it could be improved if it said "close" for users and "leave" for channels? if not thats ok.

**Comment by tarkah** [comment]
Yeah I think that makes sense! @casperstorm thoughts?

**Comment by casperstorm** [comment]
Agree, this make sense.
